.github/workflows/pytest.yml

```diff
@@ -14,7 +14,7 @@ on:
         required: false
 
       enable_profiling:
-        default: true
+        default: false
```

Comment:
It seems that profiling adds a bit of instability to the test suite. Disable it by default.